CollAction/Startup.cs

```diff
@@ -320,6 +320,16 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                      new { controller = "Projects", action = "GetStatuses" }
                  );
 
+                routes.MapRoute("MyProjects",
+                     "api/manage/myprojects",
```

Comment:
We'll need to move towards graphql I think. Too many weird "query"-like endpoints. For now, this is fine.

Reply:
I think it would be good to rebuild the whole system using a static site generator with React components (Gatsby or something like that) and a good API for the interactive parts. Would save us in hosting costs as well.